src/main/groovy/org/edx/jenkins/dsl/JenkinsPublicConstants.groovy

```diff
@@ -81,10 +81,10 @@ class JenkinsPublicConstants {
         return secretMap
     }
 
-    public static final String JENKINS_PUBLIC_JUNIT_REPORTS = 'edx-platform*/**/nosetests.xml,edx-platform*/reports/acceptance/*.xml,' +
-                                                              'edx-platform*/reports/quality.xml,edx-platform*/reports/javascript/' +
+    public static final String JENKINS_PUBLIC_JUNIT_REPORTS = '**/nosetests.xml,**/reports/acceptance/*.xml,' +
```

Comment:
Just want to double check- will the glob pattern work for both master and pr jobs? (because the filepaths will be different)

Reply:
yep, i kicked off both pr/merge jobs and everything seems to be consistent.